From 15696815c4b7e27b537adceaa1a93f414e391ae0 Mon Sep 17 00:00:00 2001
From: Ky-Anh Huynh <kyanh@theslinux.org>
Date: Thu, 17 Oct 2013 23:08:46 +0700
Subject: [PATCH] Not as said in #cff343f, we need to keep $provides variable

That allows group can use feature package instead of the origin package
---
 pacman/src/pacman-4.1.0/scripts/makepkg.sh.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in b/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in
index 1b36f74..b5fdfd2 100644
--- a/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in
+++ b/pacman/src/pacman-4.1.0/scripts/makepkg.sh.in
@@ -1818,6 +1818,7 @@ write_pkginfo() {
 
 	if [[ "$PACKAGE_CONFLICT_TYPE" == "@" ]]; then
 		conflicts=("${conflicts[@]}" "$pkgname")
+		provides=("${provides[@]}" "$pkgname")
 	fi
 
 	[[ $license ]]      && printf "license = %s\n"     "${license[@]}"
-- 
1.8.4

